.github/workflows/release.yaml

```diff
@@ -201,3 +201,69 @@ jobs:
           helm registry logout ${{ env.OCI_REGISTRY }}
         env:
           HELM_EXPERIMENTAL_OCI: "1"
+
+  publishLambdaFunction:
+    name: Publish AWS Lambda Function to SAR
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    needs: releaseBinaries
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      matrix:
+        include:
+          - arch: amd64
+            path_suffix: linux_amd64_v1
+          - arch: arm64
+            path_suffix: linux_arm64_v8.0
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ vars.AWS_ACCOUNT_ID }}:role/github
+          aws-region: ${{ vars.AWS_REGION }}
+
+      - name: Install SAM CLI
+        uses: aws-actions/setup-sam@v2
+
+      - name: Download GoReleaser artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: artifacts
+          path: dist/
+
+      - name: Prepare Lambda function for deployment
+        run: |-
+          VERSION="${GITHUB_REF#refs/tags/}"
+          cd deploy/awslambda/function
+          mkdir -p dist config-layer
+          cp ../../../dist/cerbosfunc_${{ matrix.path_suffix }}/cerbosfunc dist/bootstrap
+          chmod +x dist/bootstrap
+          cp .cerbos.yaml config-layer/
+          
+          arch=$(sed -e 's/aarch64/arm64/' -e 's/amd64/x86_64/' <<< "${{ matrix.arch }}")
+          app_name="cerbos-lambda-function-${arch/_/-}" # _ isn't valid character in the name
+
+          # Create template with architecture-specific name, replace arch (noop if arch is arm64)
+          sed -e "s/\"cerbos-lambda-function\"/\"${app_name}\"/" -e "s/- arm64/- ${arch}/" sam.yml > .sam.tmp.yml
+
+      - name: Package Lambda function
+        run: |-
+          cd deploy/awslambda/function
+          sam package \
+            --template-file .sam.tmp.yml \
+            --s3-bucket ${{ vars.AWS_SAM_PACKAGING_BUCKET }} \
+            --output-template-file packaged-template.yml \
+            --region ${{ vars.AWS_REGION }}
+      - name: Publish to AWS Serverless Application Repository
+        run: |-
+          VERSION="${GITHUB_REF#refs/tags/}"
+          cd deploy/awslambda/function
+          sam publish \
+            --template packaged-template.yml \
+            --semantic-version "${VERSION#v}" \
+            --region ${{ vars.AWS_REGION }}
```

Comment:
Some of this is already in the `justfile`. If the deployment logic is written as a   `just` recipe, we could just invoke that from the workflow with the correct arguments. That would make it easier to test and fix the script without having to run the workflow. It also gives us the ability to do a manual release if something fails for whatever reason. Otherwise we'd have to re-tag and create an entirely new release of the PDP just to trigger this workflow again.